application/controllers/admin/ExpressionValidate.php

```diff
@@ -195,6 +195,7 @@ public function email($iSurveyId, $lang)
                 $aReplacement["LASTNAME"] = gT("Participant - Last name");
                 $aReplacement["LANGUAGE"] = gT("Participant - Language");
                 $aReplacement["OPTOUTURL"] = gT("URL for a respondent to opt-out of this survey");
+                $aReplacement["GLOBALOPTOUTURL"] = gT("URL for a respondent to opt-out of surveys in this server");
```

Comment:
This sentence is false, it's «You have been removed from the central participants list for this site » the result 

not more : https://github.com/LimeSurvey/LimeSurvey/blob/ef2b9e003080274edcc888f8f012ce94b2fa74ce/application/controllers/OptoutController.php#L183

Reply:
ok, I think we understood where you are going. Will review this description.